.github/workflows/platform.check.psrule.yml

```
@@ -1,4 +1,4 @@
-name: "avm.platform.check.psrule"
+name: Platform - Check PSRule
```

If we think this to be up for discussion, what about we remove the platform workflows renaming from the PR and just prioritize the removal of BRM platform workflows, since that is already agreed?

---

Reaching out interenally to see if we can agree quickly - if not - will undo and also perform the reminaing changes.

---

In general I'm fine with the rename, as long as module workflows keep the alignment with the module name.

If we want to keep platform pipelines on top in the UI, as done in CARML as well, I'd suggest this small change
```suggestion
name: .Platform - Check PSRule
```